app/model/DTO/Participant/UpdateParticipant.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Model\DTO\Participant;
+
+class UpdateParticipant
+{
+    public const FIELD_DAYS       = 'days';
+    public const FIELD_PAYMENT    = 'payment';
+    public const FIELD_REPAYMENT  = 'repayment';
+    public const FIELD_IS_ACCOUNT = 'isAccount';
+
+    /** @var int */
+    private $eventId;
+
+    /** @var int */
+    private $participantId;
+
+    /** @var string */
+    private $field;
+
+    /** @var string */
+    private $value;
+
+    public function __construct(int $eventId, int $participantId, string $field, string $value)
```

Comment:
Tady by chtělo v budoucnu předávat všechny fieldy (vytáhnout z DB -> upravit ten jeden -> celý poslat v commandu), ale beru, že tohle je téměř 1:1 přesun do komponent, tak to můžeme upravit dodatečně.